app.js

```diff
@@ -1,12 +1,16 @@
 const express = require('express');
 const app = express();
 
-const { getCategories } = require('./controllers/games');
-
+const { getCategories, getReviews } = require('./controllers/games');
 
+app.get('/api', (req, res) => {
+    res.status(200).send( {msg: "So far so good"} )
+})
 
 app.get('/api/categories', getCategories);
 
+app.get('/api/reviews', getReviews);
+
 
 app.use((err, req,  res, next) => {
     if (err.code === '22P02') {
```

Comment:
Is this error handling required at the moment, if not please could you remove this.